sources/install.sh

```diff
@@ -1778,6 +1778,12 @@ function install_crackhound() {
   git -C /opt/tools/ clone https://github.com/trustedsec/CrackHound
 }
 
+function install_kerbrute() {
+  colorecho "Installing Kerbrute"
+  wget https://github.com/ropnop/kerbrute/releases/latest/download/kerbrute_linux_amd64 -O /usr/local/bin/kerbrute
```

Comment:
All custom binaries installed in Exegol, like this one, should be added in `/opt/tools/bin`. Could you make the change?